create/manager_triton.go

```diff
@@ -311,12 +325,49 @@ func NewTritonManager(remoteBackend backend.Backend) error {
 			},
 		}
 
-		i, _, err := prompt.Run()
-		if err != nil {
-			return err
+		continueOptions := []struct {
+			Name  string
+			Value bool
+		}{
+			{"Yes", true},
+			{"No", false},
+		}
+
+		continuePrompt := promptui.Select{
+			Label: "Attach another?",
+			Items: continueOptions,
+			Templates: &promptui.SelectTemplates{
+				Label:    "{{ . }}?",
+				Active:   fmt.Sprintf("%s {{ .Name | underline }}", promptui.IconSelect),
+				Inactive: "  {{.Name}}",
+				Selected: "  Attach another? {{.Name}}",
+			},
+		}
+
+		networksChosen := []string{}
+		shouldPrompt := true
+		for shouldPrompt {
+			// Network Prompt
+			i, _, err := networkPrompt.Run()
+			if err != nil {
+				return err
+			}
+			networksChosen = append(networksChosen, networks[i].Name)
+
+			// Remove the chosen network from the list of choices
+			networkChoices := networkPrompt.Items.([]*network.Network)
+			remainingChoices := append(networkChoices[:i], networkChoices[i+1:]...)
+			networkPrompt.Items = remainingChoices
+
+			// Continue Prompt
```

Comment:
Should we exit if there are no more remainingChoices? e.g. len(remainingChoices) == 0

Reply:
Yeah it should terminate. I'll add that check.